spec/rails_helper.rb

```diff
@@ -79,6 +79,13 @@
 
   config.include FactoryBot::Syntax::Methods
   config.include RSpec::JsonExpectations::Matchers
+
+  config.before do
```

Comment:
these stubs will apply to _all_ tests - which is probably not a good idea. Could you stub in just the individual tests as required?

Reply:
I can, it'll be over 100 files though, as a lot of the spec files have `let(:move) { create(:move...` in

Reply:
I think there 56 tests or so that broke when Adam first did this, which is quite the number to be stubbing individually. Is there a middle ground way?

Reply:
Ahhh... I see what you mean. Let me have a think about this...